app/scripts/views/mixins/password-mixin.js

```diff
@@ -44,12 +37,8 @@ define(function (require, exports, module) {
           this.logViewEvent('password.visible');
         } else {
           passwordField.attr('type', 'password');
-          if (this.isPasswordAutoCompleteDisabled()) {
-            passwordField.attr('autocomplete', 'off');
-          } else {
-            passwordField.removeAttr('autocomplete')
-              .removeAttr('autocorrect').removeAttr('autocapitalize');
-          }
+          passwordField.removeAttr('autocomplete')      
+               .removeAttr('autocorrect').removeAttr('autocapitalize');
```

Comment:
@pdehaan, I did what you said and when I ran grunt lint it gave me Warning: Task "jscs:app" failed. So, I had to undo and it came out without errors. Beginning to understand the importance of styling @vladikoff